DetDescr/python/EcalGeometry.py

```diff
@@ -110,12 +110,12 @@ def v13() :
     def v14() :
         eg = EcalGeometry(detectors_valid = ["ldmx-det-v14","ldmx-det-v14.*","ldmx-lyso-r1-v14", "ldmx-lyso-r1-v14.*"],
                 gap = 1.5,
-                layerZPositions = [ 
-                      7.932, 14.532, 32.146, 40.746, 58.110, 67.710, 86.574, 96.774, 115.638, 125.838, 
-                      144.702, 154.902, 173.766, 183.966, 202.830, 213.030, 231.894, 242.094, 260.958, 
-                      271.158, 290.022, 300.222, 319.086, 329.286, 351.650, 365.250, 387.614, 401.214, 
-                      423.578, 437.178, 459.542, 473.142, 495.506, 509.106
-                      ],
+		layerZPositions = [
+    			7.448, 16.064, 33.229, 43.202, 60.366, 71.345, 89.994, 101.581, 120.240, 131.828, 
+    			150.496, 162.072, 180.738, 192.312, 210.983, 222.563, 241.222, 252.809, 271.473, 283.052,
+    			301.710, 313.298, 331.967, 343.544, 365.708, 380.687, 402.856, 417.833, 439.992, 454.982,
+    			477.138, 492.114, 514.294, 529.266
+			],
```

Comment:
I'm guessing your editor is using tabs instead of spaces. Please add the spaces to help it line up across all editors.

Reply:
OK I did this now!